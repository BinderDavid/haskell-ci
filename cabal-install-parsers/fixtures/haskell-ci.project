-- Cabal project configuration
-- Consult http://cabal.readthedocs.io/en/latest/nix-local-build.html for more information

packages: .
optional-packages: cabal-parsers

tests: true

package haskell-ci
  ghc-options: -Wall
  ghc-options: -Werror
