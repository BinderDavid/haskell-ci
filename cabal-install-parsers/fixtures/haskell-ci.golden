Project
  {prjAllowNewer = [],
   prjConstraints = [],
   prjMaxBackjumps = Nothing,
   prjOptPackages = [],
   prjOptimization = OptimizationOn,
   prjPackages = [".", "cabal-parsers"],
   prjReorderGoals = False,
   prjSourceRepos = [],
   prjUriPackages = []}
